### DISCUSSION TITLE
Shorten HindSight Capture Time

### DISCUSSION THREAD

**Discussion by dhcrump** [gopro/labs#149] [Feature Requests]
I would like to shorten the capture time before pushing the shutter button.
I only need one to three seconds instead of the standard 15 or 30 seconds.
Any ideas or work around?

**Comment by dnewman-gpsw** [comment]
What is the application, other then avoiding some editing after capture?  I think 5 seconds might be possible, but nothing shorter was designed.  SD Card is used as buffer, so shorter times are harder than longer.

**Comment by dhcrump** [comment]
Hi David,
I'll be recording 160 cars crossing a finish line at speeds
up to 200mph. I've not had a chance to test yet but was
planning on using 1080p at 240 FPS.

There are two finish lines involved.
If I use 15 seconds pre shutter and two seconds post shutter
that will be @ 45 minutes of vid per finish line or 90 minutes total.

I'm only looking for the "instant" that the car crosses the line.
So my thought was two seconds pre and two seconds post shutter.
This would yield a bit over 10 minutes per finish line which sounds
a lot better to me.

I hope I've explained my situation well enough to get an idea why
I want a shorter HindSight capture time.


​Thanks - Dabney Crump
303-324-1084
________________________________

**Comment by ap0ll0nas** [comment]
I want 3 sec too. Often, activities are fast such that motion detection misses a good part at the start. We want 3 or 5 sec hindsight so the entire start is captured and a bit more but 15 adds work in post to go and find the actual motion start. My use case is biking over a section.

**Comment by Tjay311** [comment]
I'd actually love to see a longer.... say 60 seconds

**Reply by dnewman-gpsw** [reply]
There isn't longer hindsight, but there is a 5 second option added in the latest HERO11 Labs.  Command is hS5.

**Reply by Tjay311** [reply]
that's awesome. I could even benefit from a 45 second mode.

**Reply by dhcrump** [reply]
I'll have to check out the "hS5" command - Thanks.

**Reply by GeezeMoggers** [reply]
> There isn't longer hindsight, but there is a 5 second option added in the latest HERO11 Labs. Command is hS5.

ohhh, is this a HERO11 only thing? I have a HERO9 and would love to try this.  I'm new to this but loving what it gives. I'm using the **QR Control** tool currently would I put this code in there?

thanks

**Reply by dnewman-gpsw** [reply]
Unlikely on H9, but yes you can try with QRControl.  Just type the command into the Additional Command field
